materialize-sql/std_sql.go

```diff
@@ -292,38 +272,6 @@ func (col anyColumn) String() string {
 	return string(col)
 }
 
-// StdListTables is a convenience function for getting a list of tables for Client implementations
-// which use Go's standard `sql.DB` type and systems with an information_schema schema.
-func StdListTables(ctx context.Context, db *sql.DB, catalog string, schema string) ([]string, error) {
```

Comment:
I removed the tests that used this function; they were the ones that added more than one table at a time. These tests made the overall tests much more complicated and its trivial to test this scenario manually so it didn't seem worthwhile to keep.